src/main/scala/com/knoldus/aws/services/dynamodb/QuestionService.scala

```diff
@@ -0,0 +1,15 @@
+package com.knoldus.aws.services.dynamodb
+
+import com.knoldus.aws.models.dynamodb.{ Question, QuestionUpdate }
+
+import scala.concurrent.Future
+
+trait QuestionService {
+  def submitQuestion(question: Question): Future[Option[String]]
+
+  def getQuestion(id: String, category: String): Future[Option[Question]]
+
+  def updateQuestion(id: String, category: String, update: QuestionUpdate): Future[Option[String]]
+
+  def deleteQuestion(id: String, category: String): Future[Unit]
```

Comment:
Can we ignore returning Future[Unit]?
The method may return some result as below,
**Future[Boolean] -** If delete operation performed successfully then TRUE else FALSE [Recommended]
**Future[Int] -** Number of records removed [Recommended if multiple records are expected to be removed]

Reply:
The reason for using Future[Unit] here is because the delete method itself does not return any specific value but Unit.